grocy/rootfs/var/www/grocy/middleware/HomeAssistantAuthMiddleware.php

```
@@ -0,0 +1,128 @@
+				$db = DatabaseService::GetInstance()->GetDbConnection();
+				$user = $db->users()->where('username', $username)->fetch();
+
+				if ($user == null)
+				{
+					UsersService::GetInstance()->CreateUser($username, $username, '', '');
+					$user = $db->users()->where('username', $username)->fetch();
+				}
+
+				$stayLoggedInPermanently = isset($postParams['stay_logged_in']) && $postParams['stay_logged_in'] == 'on';
+				$sessionKey = SessionService::GetInstance()->CreateSession($user->id, $stayLoggedInPermanently);
+				self::SetSessionCookie($sessionKey);
```

PHPStan (2.1.54)

[error] Call to an undefined static method Grocy\Middleware\HomeAssistantAuthMiddleware::SetSessionCookie().


(staticMethod.notFound)